api/optimeet/groups/models.py

```diff
@@ -14,4 +17,9 @@ class Group(models.Model):
     host_id = models.CharField(max_length=64)
     max_capacity = models.IntegerField(default=5)
     status = models.CharField(max_length=1, choices=status_choices, default = ACTIVE)  
-    
+
+class UserGroup(models.Model):
+    user_id = models.CharField(max_length=64)
+    group_id = models.ForeignKey(Group, on_delete=models.CASCADE)
```

Comment:
We don't want the FE to have to pass in the `group_id` since we already have it from the URL, so let's set `default=""` so it doesn't require it. This shouldn't be an issue since we're always going to set it in the serializer's create function.